src/utils.js

```diff
@@ -59,18 +61,49 @@ export function isLinkInternal(url) {
 }
 
 // takes in a permalink and returns a URL that links to the image on the staging branch of the repo
-export function prependImageSrc(repoName, chunk) {
-  const $ = cheerio.load(chunk);
+// export function prependImageSrc(repoName, chunk) {
+//   const $ = cheerio.load(chunk);
+//   $('img').each((i, elem) => {
+//     // check for whether the original image source is from within Github or outside of Github
+//     // only modify URL if it's a permalink on the website
+//     if (isLinkInternal($(elem).attr('src'))) {
+//       $(elem).attr('src', `https://raw.githubusercontent.com/isomerpages/${repoName}/staging${$(elem).attr('src')}?raw=true`);
+//     }
+//     // change src to placeholder image if images not found
+//     $(elem).attr('onerror', "this.onerror=null; this.src='/placeholder_no_image.png';")
+//   });
+//   return $.html();
+// }
+
+export async function prependImageSrc(repoName, chunk) {
+  const $ = cheerio.load(chunk)
+  const imagePromises = []
+  const elementsToUpdate = []
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        staleTime: 1000 * 60 // 60 seconds
+      }
+    }
```

Comment:
Could we use the existing queryclient `useQueryClient` instead?

Reply:
Hooks can't be used outside of component function bodies apparently so it throws an error when `useQueryClient` is called in utils. I've stuck with creating a new `QueryClient` instance for now